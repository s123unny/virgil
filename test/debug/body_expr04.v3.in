b ../core/body_expr04.v3 8
run
bt
info l
c
q
