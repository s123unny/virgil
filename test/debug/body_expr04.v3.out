breakpoint|../core/body_expr04.v3 8|0
stopOnBreakpoint
bt|main|../core/body_expr04.v3|8
variable|0|lm|<null>:body_expr04.m|void -> int|false
variable|1|ln|<null>:body_expr04.n|void -> int|false
end
result|234
